ibc_data/conditions.csv

```diff
@@ -0,0 +1,377 @@
+MathLanguage,wordlist_visual,Read word list
+SpatialNavigation,encoding_phase,Encode location of key building
+SpatialNavigation,intersection,Participant at an intersection in the city.
+SpatialNavigation,retrieval,Retrieve location of key building
+SpatialNavigation,control,Camera approaches intersection in an control trial
+SpatialNavigation,pointing_control,Participant rotates camera to point to blue building in control trial
+SpatialNavigation,experimental,Camera approaches intersection in an experimental trial.
+SpatialNavigation,pointing_experimental,Participant rotates camera to point to key building in experimental trial
+SpatialNavigation,navigation,Camera approaches intersection during training
+EmoMem,neutral_image,Neutral background image
+EmoMem,negative_image,Negative background image
+EmoMem,positive_image,Positive background image
+EmoMem,object,Neutral object
+EmoReco,neutral_female,Neutral emotion on female face
+EmoReco,neutral_male,Neutral emotion on male face
+EmoReco,angry_female,Angry emotion on female face
+EmoReco,angry_male,Angry emotion on male face
+StopNogo,go,Arrow stays black; press button corresponding to arrow direction
+StopNogo,nogo,Arrow starts out red so do not press button
+StopNogo,successful_stop,Arrow starts out black but turns red; motor response inhibited
+StopNogo,unsuccessful_stop,Arrow starts out black but turns red; motor response not inhibited
+Catell / oddball,easy_oddball,Easy oddball trial where the non-oddball images are similar and very different from the oddball
+Catell / oddball,hard_oddball,Difficult oddball trial where all images are similar
+FingerTapping,specified,Finger to tap is highlighted
+FingerTapping,chosen,Participant chooses 1 out of 3 highlighted fingers to tap
+FingerTapping,null,No finger tap
+VSTMC,stim_load1,Dots in only one colour move coherently in a given direction
+VSTMC,stim_load2,Dots in two colours move coherently in two different directions
+VSTMC,stim_load3,Dots in all three colours move coherently in 3 different directions
+VSTMC,resp_load1,Response period of stim_load1 trials
+VSTMC,resp_load2,Response period of stim_load2 trials
+VSTMC,resp_load3,Response period of stim_load3 trials
+RewProc,green,Subject selected the green pattern
+RewProc,purple,Subject selected the purple pattern
+RewProc,left,Selected pattern was in the left side of the screen
+RewProc,right,Selected pattern was in the right side of the screen
+RewProc,switch,Selected pattern was different from the one selected in previous trial
+RewProc,stay,Selected pattern was the same as the one selected in previous trial
+RewProc,plus_20,Gained 20 units of reward as a result of the selection
+RewProc,minus_20,Lost 20 units of reward as a result of the selection
+RewProc,plus_10,Gained 10 units of reward as a result of the selection
+RewProc,minus_10,Lost 10 units of reward as a result of the selection
+NARPS,strongly_accept,Subject accepted the gamble with high confidence

```

Comment:
in NARPS we also had parametric regressors didn't we ?

Reply:
There was a condition missing, indeed. In the documentation is written as follows: << Mixed gamble stimulus with 17 units of potential gain and 15 units of potential loss (amounts could vary between 5-20) >> Is this what you meant?

Reply:
No, for NARPS we define specific parametric regressors for gain/loss. See
https://github.com/hbp-brain-charting/public_analysis_code/blob/master/ibc_public/utils_paradigm.py#L387

Reply:
Oh I see! In the documentation, this is specified as contrasts (in the contrast-definition table), not as conditions for the task.

Reply:
But as you can see, NARPS contains, 'gain', 'loss' and various 'response' conditions.
Maybe it's easier if you can visualize it. I'll try to get you a  design matrix that plots it.